examples/plot_conditional_vs_marginal_xor_data.py

```diff
@@ -0,0 +1,143 @@
+"""
+Conditional vs Marginal Importance on the XOR dataset
+==================================================================
+
+This example illustrates on XOR data that variables can be conditionally important even
+if they are not marginally important. The conditional importance is computed using the
+CPI class and the marginal importance is computed using univariate models.
+"""
+
+#############################################################################
+# To sovlve the XOR problem, we will use a SVC with RBF kernel. The decision function of
+# the fitted model shows that the model is able to separate the two classes.
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+from sklearn.base import clone
+from sklearn.linear_model import RidgeCV
+from sklearn.metrics import hinge_loss
+from sklearn.model_selection import KFold, train_test_split
+from sklearn.svm import SVC
+
+from hidimstat import CPI
+
+rng = np.random.RandomState(0)
+X = rng.randn(400, 2)
+Y = np.logical_xor(X[:, 0] > 0, X[:, 1] > 0).astype(int)
+
+xx, yy = np.meshgrid(
+    np.linspace(np.min(X[:, 0]), np.max(X[:, 0]), 100),
+    np.linspace(np.min(X[:, 1]), np.max(X[:, 1]), 100),
+)
+
+X_train, X_test, y_train, y_test = train_test_split(X, Y, test_size=0.2, random_state=0)
+model = SVC(kernel="rbf", random_state=0)
+model.fit(X_train, y_train)
+
+Z = model.decision_function(np.c_[xx.ravel(), yy.ravel()])
+fig, ax = plt.subplots()
+
+ax.imshow(
```

Comment:
If you want to do this plot for explain the data. In my opinion, it's better to use a density plot (see the function: https://seaborn.pydata.org/generated/seaborn.kdeplot.html).

Reply:
I thought about it initially but agreed that level lines don't add much information and incur additional computation costs. The point is to illustrate that the SCV solves the classification task visually I thought imshow is good enough for that. WDYT?

Reply:
I prefer to have the explanation attached to the figure that before the code.
If you can add a name for this section, it will be nice.

Reply:
done